GRDB/Core/Database.swift

```diff
@@ -1237,8 +1237,29 @@ extension Database {
     ///         try db.usePassphrase("secret")
     ///     }
     public func usePassphrase(_ passphrase: String) throws {
-        let data = passphrase.data(using: .utf8)!
-        let code = data.withUnsafeBytes {
+        guard !passphrase.isEmpty, var data = passphrase.data(using: .utf8) else {
+            throw DatabaseError(message: "usePassphrase Failed: invalid or empty passphrase")
```

Comment:
Please don't quote the method name in the error message. This leads to copy-paste errors (see `changePassphrase` below). And this is not the common practice in this repo. `invalid or empty passphrase` is enough.